Interactive Plugin/build/resources/main/dialogs.xml

```diff
@@ -247,6 +243,8 @@
         <step>7</step>
         <body>To query purchases and obtain more details about them you must implement the skuDetailsResponseListener and call the querySkuDetailsAsync method. The automatic changes will add these methods to the main activity. If you implement the changes manually, please notice that the AppCoinsBillingStateListener uses callSkuDetails(), which is the method that calls querySkuDetailsAsync.</body>
         <body>SkuDetailsResponseListener</body>
+        <body>After starting the connection, you can query Catappult for the products available to buy in order to display them to the user. This query includes not only the product's title but also the description, value, etc...
+            To query the products you can use the querySkuDetailsAsync which requires a SkuDetailsResponseListener to process Catappult's response.F</body>
```

Comment:
I don't know if it is a misspelling or not but "response.F". F is probably extra here